enterprise/api/v1/views/enterprise_group.py

```diff
@@ -157,18 +172,34 @@ def assign_learners(self, request, group_uuid):
             customer = group.enterprise_customer
         except models.EnterpriseGroup.DoesNotExist as exc:
             raise Http404 from exc
-        if requested_emails := request.POST.dict().get('learner_emails'):
+
+        if requested_emails := request.data.get('learner_emails'):
```

Comment:
we need learner emails here, we probably want to return an error 400 if they aren't a part of the request data

Reply:
the 400 error is returned in the last line of this method
`return Response(data="Error: missing request data: `learner_emails`.", status=400)`